honest_forests/estimators/forest.py

```diff
@@ -446,7 +447,9 @@ def _predict_proba(self, X, indices=None, impute_missing=None):
         Parallel(
             n_jobs=n_jobs,
             verbose=self.verbose,
-            **_joblib_parallel_args(require="sharedmem")
```

Comment:
Amazing, thanks for this fix. An alternative fix would be to require a lower version of sklearn but this is better. I assume I had a lower version. I assume this requires sklearn > 1.0? Can you add this to the requirments.txt file?

Reply:
I believe it does require sklearn > 1.0. Note that this is because sklearn ppl changed their util.py to bump joblib version dependency to 1.0.0.

Reply:
okay. We should make sure the requirements.txt file states the proper requirements. We don't want errors because this code isn't compatible with old versions of joblib or sklearn permitted by our requirements.txt file